filters/ThirdParty/filter_253_IndianList/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 253,
   "name": "IndianList",
-  "description": "Additional filter list for websites in Indian.",
+  "description": "Additional filter list for websites in Hindi, Bengali and Tamil.",
```

Comment:
maybe it would be better to add "and other Indic languages", since we found out there are also other languages in this group. We could take out the Tamil language and add: "Additional filter list for websites in Hindi, Bengali and other Indic languages."